Exiled.API/Features/CustomStats/CustomHumeShieldStat.cs

```diff
@@ -0,0 +1,73 @@
+﻿// -----------------------------------------------------------------------
+// <copyright file="CustomHumeShieldStat.cs" company="Exiled Team">
+// Copyright (c) Exiled Team. All rights reserved.
+// Licensed under the CC BY-SA 3.0 license.
+// </copyright>
+// -----------------------------------------------------------------------
+
+namespace Exiled.API.Features.CustomStats
+{
+    using Mirror;
+    using PlayerRoles.PlayableScps.HumeShield;
+    using PlayerStatsSystem;
+    using UnityEngine;
+    using Utils.Networking;
+
+    /// <summary>
+    /// A custom version of <see cref="HumeShieldStat"/> which allows the player's max amount of HumeShield to be changed.
+    /// </summary>
+    public class CustomHumeShieldStat : HumeShieldStat
```

Comment:
Is it possible to change other properties of HumeShield?

Reply:
![image](https://github.com/Exiled-Team/EXILED/assets/85962933/ee1c03eb-8d10-4e45-8d92-d02d8c2d9f61)